### ISSUE TITLE
File not found vs module not found

### ISSUE THREAD

**Issue by rogeliog** [geowarin/friendly-errors-webpack-plugin#5]
I have this code

```
import something from '../path/to/file';
import redux from 'redux';
```

But the error output is the following

```
These dependencies were not found in node_modules:

* ../path/to/file
* redux

Did you forget to run npm install --save for them?
```

It would be great if we could split this into two separate type of errors
### Module not found

```
This dependency was not found in node_modules:

* redux

Did you forget to run npm install --save for it?
```
### File not found

```
This file or directory as was not found

* ../path/to/file

```

For the file not found it would be quite interesting to search for that file in a one directory above and one directory below. For example for `../path/to/file` we could check in  `./path/to/file` and `../../path/to/file`. If we do something like this, then we could add that info to the output.

```
This file or directory as was not found

* ../path/to/file

Did you mean `../../path/to/file`?
```

**Comment by geowarin** [comment]
That's a great idea !
Why not using a running a glob on all sources (if we can determine we the sources are located)?
Would you be willing to submit a PR for this?

**Comment by rogeliog** [comment]
Yeah! I'm going to be a bit busy for the first half of the week, but I'll work on it as soon as I have some time :)

**Comment by guox191** [comment]
+1, It's more precise

**Comment by geowarin** [comment]
Released in 1.6.0
